backend/LeetWars.Core/LeetWars.Core.DAL/Entities/Friendship.cs

```diff
@@ -0,0 +1,21 @@
+﻿using LeetWars.Core.DAL.Enums;
+
+namespace LeetWars.Core.DAL.Entities
+{
+    public class Friendship : Entity<long>
+    {
+        public DateTime CreatedAt { get; set; }
```

Comment:
why not Audit?

Reply:
Anton and I came to the conclusion that this is the database scheme that will allow us to retrieve all the necessary data without violating the rules of normalization